include/morphio/vasc/properties.h

```diff
@@ -68,7 +69,7 @@ struct Properties {
     std::vector<Connection::Type> _connectivity;
 
     template <typename T>
-    std::vector<typename T::Type>& get() noexcept;
+    std::vector<typename T::Type>& get_mut() noexcept;
```

Comment:
wouldn't be clearer to have : 
```
get
cget (or get_const)
```
as names instead ? I feel like the `mut` suffix collides a bit with the "mutable morphologies" naming convention. 

But this is maybe personal and a question of taste.

Reply:
`const` by default seems like a better interface…